### ISSUE TITLE
Warning: Non-portable path to file "BranchViewHandler.h"

### ISSUE THREAD

**Issue by SSheldon** [BranchMetrics/ios-branch-deep-linking#559]
Xcode 8.3 introduces new warning checks, including `nonportable-include-path` which warns when imports use different case than the actual files.

This warning is triggered in the BranchViewHandler.m file:
```
BranchViewHandler.m:11:9: Non-portable path to file '"BranchViewHandler.h"'; specified path differs in case from file name on disk
```

Looks like it's because the import of the header file uses a lowercase h, I'm assuming this fix is just:
``` diff
-#import "BranchViewhandler.h"
+#import "BranchViewHandler.h"
```

**Comment by E-B-Smith** [comment]
Thank you!  That is it.  It's easy to miss this on the OS X case-insensitive files systems.

**Comment by E-B-Smith** [comment]
Patched with PR #560.
